README.md

```diff
@@ -103,6 +103,7 @@ Ubuntu 20.04 LTS (Focal Fossa):
 ## Documentation
 
 * [Okapi Guide and Reference](doc/guide.md)
+* API: [RAML and schemas](okapi-core/src/main/raml) [dev.folio.org](https://dev.folio.org/reference/api/#okapi)
```

Comment:
The rendered README blurs those links together. Better would be\
`API: [RAML and schemas](okapi-core/src/main/raml) and generated [API documentation](https://dev.folio.org/reference/api/#okapi)`